packages/datadog-instrumentations/src/cucumber.js

```diff
@@ -747,6 +750,9 @@ function getWrappedRunTestCase (runTestCaseFunction, isNewerCucumberVersion = fa
 
 function getWrappedParseWorkerMessage (parseWorkerMessageFunction, isNewVersion) {
   return function (worker, message) {
+    if (!testSuiteFinishCh.hasSubscribers) {
+      return parseWorkerMessageFunction.apply(this, arguments)
+    }
```

Comment:
unrelated changes, but disabling the cypress plugin caused issues in the instrumentation. It's always recommended to check `.hasSubscribers` before doing logic anyway, so this was a good thing